src/numbers/realnum.rs

```diff
@@ -66,7 +67,7 @@ impl RealNumber for f64 {
     }
 
     fn rand() -> f64 {
-        let mut small_rng = SmallRng::from_seed(SEEDS);
+        let mut small_rng = get_rng_impl(Some(100101010));
```

Comment:
I think that if we pass `None` here, we will get a random seed for std targets. We could change `get_rng_impl` to get a seed from current time for other targets maybe?